docs/tests/code-coverage.md

```diff
@@ -6,8 +6,8 @@ sidebar_position: 4
 
 ## Code Climate
 
-We are using [Code Climate](https://codeclimate.com/github/forem/forem) to track
-code quality and coverage.
+We are using [Code Climate](https://codeclimate.com/) to track
+code quality and coverage. You can view our instance [here](https://codeclimate.com/github/forem/forem).
```

Comment:
I saw this PR get merged in -- It looks like we removed it as we aren't using it?https://github.com/forem/forem/pull/18867. However...do we want to use it? Did we find value?

Reply:
Hmmm weird.... I'm seeing this comment linked here but in the PR code section I'm seeing that code climate section correctly removed. I'd removed it in this [commit ](https://github.com/forem/forem-docs/pull/95/commits/c81353d6cae4ed35ff4bb04e9a93b0c6c4572dfa). Perhaps this comment was on an outdated diff?